StackOverflow/Model/DataFetchDelegate.h

```diff
@@ -0,0 +1,16 @@
+//
+//  DataFetchDelegate.h
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol DataFetchDelegate <NSObject>
+
+- (void)didReceiveData:(NSArray *)groups;
```

Comment:
Why is the method called didReceiveData: when in fact it's retiring groups ? Surely it should be called didReceiveQuestionGroups ?